src/ruleFinder/IssueCommentsRuleFinder.js

```diff
@@ -52,14 +52,19 @@ module.exports = class IssueCommentsRuleFinder {
     const rules = [];
 
     if (!Utils.contextHasAction(context, 'created')) {
-      return [];
+      return rules;
     }
 
     const {issue} = context.payload;
     const repositoryConfig = this.configProvider.getRepositoryConfigFromIssue(this.config, issue);
+
+    if (Utils.issueHasLabel(issue, repositoryConfig.labels.waitingAuthor.name)) {
```

Comment:
Mmmh, shouldn't we verify if the comment has been added from the author?

Reply:
we want to remove the label whoever wrote the comment (could be the author, anyone), at least that's what was asked, sometimes someone else gives an information that will unlock the PR

that way the issue will appear in the QA's filter: `is:open is:issue label:NMI -label:"Waiting for author"`

Reply:
So, what if I add the label before commenting? :trollface:

Reply:
I checked with them, we should also check that the NMI label is there too